FormType/NewThreadMultipleMessageFormType.php

```diff
@@ -14,12 +16,12 @@ class NewThreadMultipleMessageFormType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('recipients', 'recipients_selector', array('label' => 'recipients', 'translation_domain' => 'FOSMessageBundle'))
-            ->add('subject', 'text', array('label' => 'subject', 'translation_domain' => 'FOSMessageBundle'))
-            ->add('body', 'textarea', array('label' => 'body', 'translation_domain' => 'FOSMessageBundle'));
+            ->add('recipients', RecipientsType::class, array('label' => 'recipients', 'translation_domain' => 'FOSMessageBundle'))
```

Comment:
Should be replaced by something like https://github.com/FriendsOfSymfony/FOSUserBundle/blob/master/Util/LegacyFormHelper.php